etc/p3-config/p3-config.yml

```diff
@@ -633,6 +946,25 @@ p3_images:
   - "{{ p3_image_anaconda }}"
   - "{{ p3_image_fedora_atomic }}"
   - "{{ p3_image_fedora_coreos }}"
+  - "{{ p3_image_ubuntu_18_04 }}"
+
+# Ubuntu 18.04
+p3_image_ubuntu_18_04:
+  name: "ubuntu-18.04"
+  image_url: "https://cloud-images.ubuntu.com/bionic/current/bionic-server-cloudimg-amd64.img"
+  properties:
+    os_distro: "ubuntu"
+    os_version: "18.04"
+    hw_pci_numa_affinity_policy: required
+    hw_machine_type: q35
+    hw_disk_bus: scsi
+    hw_scsi_model: virtio-scsi
+    hw_cdrom_bus: scsi
+    hw_rescue_bus: sata
+    hw_video_model: virtio
+    hw_serial_port_count: 0
+    hw_vif_multiqueue_enabled: true
+    hw_emulator_threads_policy : shared
```

Comment:
I am tempted to say we should leave the emulator threads off here, just let the flavor do it?